python/tests/test_account.py

```diff
@@ -220,6 +220,18 @@ def test_get_set_profile_image_simple(self, ac1, data):
         chat.remove_profile_image()
         assert chat.get_profile_image() is None
 
+    def test_mute(self, ac1):
+        chat = ac1.create_group_chat(name="title1")
+        assert not chat.is_muted()
+        assert chat.mute()
+        assert chat.is_muted()
+        assert chat.unmute()
+        assert not chat.is_muted()
+        assert chat.mute(50)
+        assert chat.is_muted()
+        assert chat.mute(-51)
```

Comment:
This call should raise an exception now I think, as the FFI API returns an error for this.  Can be tested like:
```
with pytest.raises(ValueError):
    chat.mute(-51)
```
And the implementation would then have to raise ValueError of course.